src/feed/Feed.js

```diff
@@ -73,25 +72,25 @@ class Feed {
         let response = await tickHistoryPromise;
 
         // Clear all notifications related to active symbols
-        this._mainStore.notification.removeByCategory('activesymbol');
+        // this._mainStore.notification.removeByCategory('activesymbol');
```

Comment:
just remove it. Leaving it will confuse people in the future.

Btw this code was added for a reason. Ashkan wanted all active symbols related notifications to be cleared when the symbol is changed. To do this in `app` subscribe to `onSymbolChanged` event and clear those notifications by category.